src/App.js

```diff
@@ -1,20 +1,28 @@
 import React from 'react';
 import TodoList from './components/TodoComponents/TodoList';
 import TodoForm from './components/TodoComponents/TodoForm';
+import './App.css';
 //data
 const TodoData = [
   {
     task: 'Organize Garage',
-    id: 1528817077286,
+    id: 1,
     completed: false
   },
   {
     task: 'Bake Cookies',
-    id: 1528817084358,
+    id: 2,
     completed: false
   }
 ];
 
+let nextId = 2;
+
+const getNewId = () => {
+  nextId += 1;
+  return nextId;
```

Comment:
You could shorten this with the pre-increment operator (which will increment a variable and evaluate to the new value, just like `numVar += 1`):
```javascript
const getNewId = () => ++nextId;
```